compiler/z/codegen/OMRMemoryReference.cpp

```diff
@@ -2527,21 +2493,11 @@ OMR::Z::MemoryReference::getSnippet()
       {
       self()->setMemRefAndGetUnresolvedData(snippet);
       }
-   else // add writable data snippet address relocation
-   if (self()->getConstantDataSnippet() != NULL)
+   // add writable data snippet address relocation
```

Comment:
nit: shouldn't comment be inside the conditional?

Reply:
I don't think these comments should exist at all. They do not provide any additional information than can be implied by the source code lines following them. Fixed in 9133acf